### PR TITLE
Improve trust relationship docs

### PR DESCRIPTION
**Pull request by meffij**

### CODE REVIEW

File: docs/Technical Reference.md [hunk 1]
```diff
@@ -141,69 +141,55 @@ A **bucket config**, meanwhile, is a JSON object that describes metadata associa
...
-        "Action": "sts:AssumeRole"
-    }
-    ]
+    "...": "whatever was here before"
+  ]
 }
 ```
 
-You can now configure a Quilt Role for this role (using e.g. `t4.create_role`).
+You can now configure a Quilt Role with this role (using the Navigator's admin panel, or `t4.admin.create_role`).
```
Comment (ResidentMario) [inline docs/Technical Reference.md]: We don't call the catalog `Navigator` anywhere anymore. Should be "using the Catalog's admin panel".

File: docs/Technical Reference.md [hunk 2]
```diff
@@ -141,69 +141,55 @@ A **bucket config**, meanwhile, is a JSON object that describes metadata associa
...
-        "Effect": "Allow",
-        "Principal": {
-        "Service": "ec2.amazonaws.com"
-        },
-        "Action": "sts:AssumeRole"
-    }
-    ]
+  "Version": "2012-10-17",
+  "Statement": [
+    "...": "one or more statements"
```
Comment (ResidentMario) [inline docs/Technical Reference.md]: This isn't correct syntax, you're missing the opening `{` and closing `}` brackets.

File: docs/Technical Reference.md [hunk 3]
```diff
@@ -141,69 +141,55 @@ A **bucket config**, meanwhile, is a JSON object that describes metadata associa
 
 A bucket config can be included inline in a federation, or it can be a standalone JSON file that is linked from a federation.
 
-## Adding users to a catalog using AWS Console
+## Preparing an AWS Role for use with T4
 
-These instructions document how to add users to your catalog instance using the AWS Console.
+These instructions document how to set up an existing role for use with T4. If the role you want to use doesn't exist yet, create it now.
```
Comment (stevededalus) [inline docs/Technical Reference.md]: Can you recommend a good default?

Comment (meffij) [reply docs/Technical Reference.md]: not really

Comment (stevededalus) [reply docs/Technical Reference.md]: I'm going to test `S3FullAccess` and see how that goes

**Pull request merged**